scss/_tables.scss

```diff
@@ -3,14 +3,15 @@
 //
 
 .table {
-  --bs-table-bg: #{$table-bg};
-  --bs-table-accent-bg: transparent;
-  --bs-table-striped-color: #{$table-striped-color};
-  --bs-table-striped-bg: #{$table-striped-bg};
-  --bs-table-active-color: #{$table-active-color};
-  --bs-table-active-bg: #{$table-active-bg};
-  --bs-table-hover-color: #{$table-hover-color};
-  --bs-table-hover-bg: #{$table-hover-bg};
+  --bs-table-bg: transparent;
+  --bs-table-color: #{$body-color};
+  --bs-table-border-color: #{$border-color};
+  --bs-table-striped-bg: #{rgba($black, $table-striped-bg-factor)};
+  --bs-table-striped-color: #{$table-color};
+  --bs-table-active-bg: #{rgba($black, $table-active-bg-factor)};
+  --bs-table-active-color: #{$table-color};
+  --bs-table-hover-bg: #{rgba($black, $table-hover-bg-factor)};
```

Comment:
We can move this to `mixins/_table-variants.scss` file and enable opt-out from declaring these values